js/samples/06.auth.botSSO/src/index.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Just FYI - I think the 06 slot got nabbed by Steve for agents. We will need to do a samples refactor anyway, so no need to make a new Auth folder unless you have the extra time.

Reply:
i'll change it to 07